src/tests/EdFi.DataManagementService.Tests.E2E/StepDefinitions/StepDefinitions.cs

```diff
@@ -302,11 +302,22 @@ public void ThenTheResponseBodyIs(string expectedBody)
 
             _logger.log.Information(responseJson.ToString());
 
-            responseJson.Should().BeEquivalentTo(expectedBodyJson, options => options
-                .WithoutStrictOrdering()
-                .IgnoringCyclicReferences()
-                .Excluding(x => x.Path.EndsWith("correlationId"))
-            );
+            (responseJson as JsonObject)?.Remove("correlationId");
+            (expectedBodyJson as JsonObject)?.Remove("correlationId");
+
+            try
+            {
+                responseJson.Should().BeEquivalentTo(expectedBodyJson, options => options
+                    .WithoutStrictOrdering()
+                    .IgnoringCyclicReferences()
+                    .RespectingRuntimeTypes()
```

Comment:
Testing I found that it always gave the evaluation of the expected result as a false positive, I found that adding `.RespectingRuntimeTypes()` solves the problem.